backports/py3-qt5.v3.14/ENABLE_ON

```diff
@@ -0,0 +1 @@
+v3.11
```

Comment:
Should we also add `v3.14`?

Reply:
On 3.14, py3-qt5 in official Alpine repository can be used. I'll fix the directory name